superset/superset_typing.py

```diff
@@ -55,6 +55,8 @@ class AdhocColumn(TypedDict, total=False):
     hasCustomLabel: Optional[bool]
     label: Optional[str]
     sqlExpression: Optional[str]
+    is_axis: Optional[bool]
+    time_grain: Optional[str]
```

Comment:
nit: these are snake_case, while the old ones are camelCase. It's a matter of taste, of course, but I kinda prefer using camelCase as these kind of originate from the TS codebase.